README.md

```diff
@@ -62,3 +70,5 @@ You want to contribute? Great! Here are the things you should do before submitti
 1. Create pull-request from your branch.
 
 That's it! :)
+
+
```

Comment:
👀

Reply:
Old habits die hard.  I dunno where I picked it up but I always have a few extra blank lines at the bottom of files.